.dockerignore

```diff
@@ -0,0 +1 @@
+venv
```

Comment:
@saady this would be required on `jekyll` otherwise it will fail since this folder provided by circleci at build time